release_notes.md

```diff
@@ -251,6 +251,7 @@ use `CodecPipelinesAdapter` manually.
 1. [TH2-3003] Added automatic attachment of example.py code in readme.md.
 2. [TH2-3558] Added more debug info about Data cache using.
 3. [TH2-3389] GetXById http-provider command handles 404 error status instead of JsonDecodeException.
+4. [TH2-3732] Log message in http.GetMessages contains name of the stream.
```

Comment:
It's not a feature. You were fixed a bug.

Reply:
My bad, i fixed it in the commit below.